modules/engine/src/main/scala/com/gsk/kg/engine/functions/Literals.scala

```diff
@@ -2,12 +2,18 @@ package com.gsk.kg.engine.functions
 
 import org.apache.spark.sql.Column
 import org.apache.spark.sql.functions.{concat => cc, _}
+import org.apache.spark.sql.types.BooleanType
 import org.apache.spark.sql.types.StringType
 
+import com.gsk.kg.engine.RdfFormatter
+
 object Literals {
 
   // scalastyle:off
-  val nullLiteral = lit(null)
+  val nullLiteral: Column    = lit(null)
```

Comment:
as this is also a constant I suggest refactoring to uppercase and maintain homogeneity.

```suggestion
  val NullLiteral: Column    = lit(null) // scalastyle:ignore
```

Also I think there is no need disable scalastyle on every constant as it only complains about `null`.